app/init.php

```diff
@@ -546,6 +548,7 @@ function (mixed $value, Document $document, Database $database) {
             $database
             ->find('subscribers', [
                 Query::equal('topicInternalId', [$document->getInternalId()]),
+                Query::limit(1000000)
```

Comment:
Let's add a const for this, `APP_LIMIT_SUBSCRIBERS`

Reply:
done this